CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 * Added validation to **validate** for release notes config file.
 * Added **isoversize** and **isautoswitchedtoquietmode** fields to the playbook schema.
 * Added to the **update-release-notes** command `-bc` flag to generate template for breaking changes version.
+* Added support for **init** command to run from non-content repo.
```

Comment:
move this to the changelog header please

Reply:
done